src/screens/login-screen.tsx

```diff
@@ -18,14 +14,27 @@ import {
 import { useLoginMutation } from "../hooks/use-login-mutation";
 import { Navigation } from "react-native-navigation";
 import { storage } from "../../utils/storage";
+import { MainButton } from "../components/main-button";
+import { StyledHeader } from "../components/header";
+import { InputSimple } from "../components/input-simple";
+import { MainContainer } from "../components/main-container";
 
 interface LoginScreenProps {
   componentId: string;
 }
 
+interface Input extends InputValidation {
+  value: string;
+}
+
+const DefaultInput: Input = {
```

Comment:
constantes nao precisam dessa primeira maiuscula
```suggestion
const defaultInput: Input = {
```